valhalla/packages/jawn/src/index.ts

```diff
@@ -0,0 +1,95 @@
+// src/index.ts
+import express from "express";
+import morgan from "morgan";
+import { Client } from "pg";
+import { hello } from "helicone-shared-ts";
+
+const app = express();
+const environment = process.env.ENV || "development";
+
+// for logs
+app.use(morgan("combined"));
```

Comment:
Are we going to hookup morgan logs with another service like datadog etc?

Reply:
yeah or cloudwatch